src/Servers/IIS/IIS/src/Core/IISHttpContextOfT.cs

```diff
@@ -82,9 +90,8 @@ public override async Task<bool> ProcessRequestAsync()
             }
             else if (!HasResponseStarted && _requestRejectedException == null)
```

Comment:
I think this case is still ambiguous, it doesn't distinguish between the client disconnecting and the server aborting. 499 should only be attributed to the client. Is there other state we can check here to distinguish between server aborted and client disconnected?

Reply:
Can we use "((ex is OperationCanceledException || ex is IOException) && _requestAborted)" condition in Or ?

Example : else if (!HasResponseStarted && _requestRejectedException == null  || ((ex is OperationCanceledException || ex is IOException) && _requestAborted)))